addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -989,6 +995,20 @@ export class Wysiwyg extends Component {
         this.el = $el[0];
         this.$el = $el;
     }
+    /**
+     * @private
+     */
+    _triggerDynamicElementWidgetsStartRequest() {
+        const dynamic_element = this.$root[0].querySelectorAll(".s_dynamic");
+        if (dynamic_element.length) {
```

Comment:
```suggestion
        if (dynamic_element.length) {
```
By nature for loop won't be execute if lenght == 0, so no need of this if check.